package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "bookstore",
+  "description": "Bookstore is a react project that enable users to create books and can also make comments. This project also leverage the use of redux.",
+  "author": {
+    "name": "Muhammad Sesay",
+    "email": "contact@maej.dev",
+    "url": "https://github.com/mmsesay"
+  },
+  "license": "MIT",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
- [ ] `react-redux is not listed among your project dependencies. Kindly add it as it will be needed to use`Redux`

Reply:
Thank you for picking it. I have installed it.